app/hotels/src/filter/stars/StarsCheckbox.js

```diff
@@ -26,9 +26,12 @@ export default function StarsCheckbox(props: Props) {
           ) : (
             <Text style={styles.text}>{props.text}</Text>
           )}
+          {/*
+          Do not show hotels numbers for now
```

Comment:
Delete it completely, please.

Reply:
Just this comment or any mentions of hotels numbers?

Reply:
I would remove all the code related to the "XYZ hotels" sentence because:

1) I don't know how to fetch it now (not that important to be honest)
2) it will become obsolete very quickly if just commented

Reply:
Done